torch/jit/__init__.py

```diff
@@ -804,3 +806,165 @@ def forward(self, *args, **kwargs):
 
 if not torch._C._jit_init():
     raise RuntimeError("JIT initialization failed")
+
+
+def _scalar(x):
+    """Convert a scalar tensor into a Python value."""
+    assert x.numel() == 1
+    return x[0]
+
+
+def _is_onnx_list(value):
```

Comment:
Why did those functions migrate from `torch/onnx` to `torch/jit`? What was wrong? I'd rather avoid keeping things like `_is_onnx_list` here.

Reply:
This was required during my early investigation since the monkey-patched .op() method was not present on `Graph`. Let me see if i can revert this move now that I have a (slightly different from my original) solution